kratos/tests/cpp_tests/sources/test_kratos_parameters.cpp

```diff
@@ -559,6 +559,7 @@ KRATOS_TEST_CASE_IN_SUITE(KratosParametersIsMethods, KratosCoreFastSuite)
         "double_value": 2.0, // This is comment too, but using another comment
         "bool_value" : true, // This is another comment being meta as realizing that all the possibilities are already check
         "string_value" : "hello",/* This is a nihilist comment about the futile existence of the previous comment as a metacomment */
+        "s_array_value" : ["hello", "world"],
```

Comment:
I have used `"s_array_value"` because with `"string_array_value"` the test was failing. Indeed, when `key` is `"string_array_value"`, these lines below are executed and the wrong method is called (thus the test is failing)

https://github.com/KratosMultiphysics/Kratos/blob/5278a666891eaee42f61ac20d2c02aea45ebb5d7/kratos/tests/cpp_tests/sources/test_kratos_parameters.cpp#L588-L589

Reply:
I have not included this suggestion in #11200 because of the comment above, ok?